packages/lib-classifier/src/store/utils/getBearerToken.js

```diff
@@ -1,10 +1,10 @@
 export default async function getBearerToken (authClient) {
   if (authClient) {
-    const authToken = await authClient.getBearerToken() || {}
-    if (authToken && authToken.token) {
-      return `Bearer ${authToken.token}`
+    const authToken = await authClient.checkBearerToken() || ''
+    if (authToken && authToken.access_token) {
```

Comment:
I think `auth.checkBearerToken()` yields a string, not an object.

Reply:
That's what I thought too, but it is returning an object. At least the oauth client does. I'll check the behavior of the first party auth client. It was a mistake for me to change the default be string if it's returning an object.

Reply:
That's confusing. `auth.checkBearerToken()` yields a string for me in #437 (and is used that way earlier in this PR too.)
https://github.com/zooniverse/front-end-monorepo/blob/7ef5dd36d4be46b44c102a4aa59a55f3efc3faf9/packages/app-project/stores/Collections.js#L62-L63

Reply:
Yep, I'm thinking it's a difference between auth and oauth cilents. Even though they have the same API method, it's returning different things.

Reply:
I've confirmed that the auth client (first party, being used by app-project) returns a string of just the bearer token. The same method for the oauth client returns an object which includes the bearer token and expiration and the token type. So, given that the classifier is meant to be consumable by either, I'm going to have to write the helper to handle either for now.

Reply:
I remember now. First party handles token refresh for you, and always returns a valid token. Third party expects you to manage expired tokens, so returns an object including the expiration info.